pallets/order-book/src/lib.rs

```diff
@@ -346,14 +350,23 @@ pub mod pallet {
             order_id: T::OrderId,
             owner_id: T::AccountId,
             side: PriceVariant,
+            price: OrderPrice,
             amount: OrderAmount,
         },
 
+        /// All amount of the limit order is executed
+        LimitOrderFilled {
```

Comment:
`LimitOrderFulfilled` seems more accurate/readable, do you agree?

Reply:
Not. `Filled` is a common status for different exchanges